airbyte-db/src/main/java/io/airbyte/db/DatabaseHelper.java

```diff
@@ -33,39 +33,46 @@
 
 public class DatabaseHelper {
```

Comment:
I wanted to reuse this class but it made assumptions that we were always using postgres. Augmented it so that it just defaults to postgres but can handle other drivers and dialects. If people hate this and would like the integration helpers to be separate and stay outside of code modules, lmk and I can adjust.